creator/ingest_runs/tasks/ingest_run.py

```diff
@@ -31,28 +31,33 @@ def run_ingest(ingest_run_uuid=None):
     are_gwo = all(check_gwo(version) for version in versions)
     if are_gwo:
```

Comment:
Could this block of code be refactored slightly so you have only one place where you're calling ingest_run.fail?

```python
        try:
            if are_gwo:
                if not settings.FEAT_INGEST_GENOMIC_WORKFLOW_OUTPUTS:
                    raise Exception(.. the feature not enable msg ..)
                else:
                    ingest_genomic_workflow_output_manifests(ingest_run)
            else:
                raise Exception(...the unknown file msg..)
        except Exception as e:
            ingest_run.fail(str(e))
            ingest_run.save()
            raise
```